tests/unit_tests/test_image_operations.py

```diff
@@ -1,15 +1,19 @@
 import unittest, math
 from SUASSystem import Location, GCSSettings, inverse_haversine
```

Comment:
Something I forgot to address earlier: PEP states that you should have separate lines for each import.